crates/client/rpc/src/lib.rs

```diff
@@ -179,11 +179,11 @@ where
         })?;
 
         let runtime_api = self.client.runtime_api();
-        let hex_address = contract_address.to_bytes_be();
-        let hex_key = H256::from_slice(&key.to_bytes_be()[..32]);
+        let hex_address = Felt252Wrapper::from(contract_address);
```

Comment:
similar to above, the `contract_address` is a `FieldElement` so you could remove this line and keep the `.into` in the `get_storage_at` call